website/docs/installing-podman-desktop/alternate-installation-methods.md

```diff
@@ -0,0 +1,235 @@
+---
+sidebar_position: 2
+title: Alternate installation methods.
+description: Consider installing Podman Desktop using one of the alternate installation method when you cannot use the default installation method.
+keywords: [podman desktop, containers, podman, installing, installation]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Installing Podman Desktop with alternate installation methods
+
+Consider using one of these alternate installation method when you cannot use the [default installation method](/docs/installing-podman-desktop).
+
+<Tabs groupId="operating-systems">
+<TabItem value="win" label="Windows silent">
```

Comment:
This is just too many tabs - it's hard to see what options are available for your OS, scan for one you're expecting, and is getting cropped in the screenshots.

Maybe this page should have top tabs for Windows/Mac/Linux too and then these are sub-tabs? I don't like double-tabs, but it would at least provide shorter lists and start by filtering to the current platform.

Reply:
I see what you mean. I'll try my best.

Reply:
I believe nested tabs is not an option in Docusaurus. https://docusaurus.io/docs/markdown-features/tabs

Reply:
Yes we can. Done.